test/PlatformTerms.js

```diff
@@ -49,5 +49,57 @@ contract("PlatformTerms", ([_, admin]) => {
     expect(feeAmount).to.be.bignumber.eq(amountInt.mul(fee.div(Q18)).round(0, 4));
   });
 
-  it("should calculate neumark share");
+  it("should calculate neumark share when reward is 0 wei", async () => {
+    const reward = 0;
+    const [platformShares, shares] = await platformTerms.calculateNeumarkDistribution(reward);
+
+    expect(shares.plus(platformShares)).to.be.bignumber.eq(reward);
+    expect(shares).to.be.bignumber.eq(0);
+    expect(platformShares).to.be.bignumber.eq(0);
+  });
+  it("should calculate neumark share when reward is 1 wei", async () => {
+    const reward = 1;
+    const [platformShares, shares] = await platformTerms.calculateNeumarkDistribution(1);
+
+    expect(shares.plus(platformShares)).to.be.bignumber.eq(reward);
+    expect(shares).to.be.bignumber.eq(1);
+    expect(platformShares).to.be.bignumber.eq(0);
+  });
+
+  it("should calculate neumark share when reward is 1 ether", async () => {
+    const reward = etherToWei(1);
+    const [platformShares, shares] = await platformTerms.calculateNeumarkDistribution(reward);
+
+    expect(shares.plus(platformShares)).to.be.bignumber.eq(reward);
+    expect(shares).to.be.bignumber.eq(etherToWei(0.5));
+    expect(platformShares).to.be.bignumber.eq(etherToWei(0.5));
+  });
+
+  it("should calculate neumark share when reward is undevisible", async () => {
+    const reward = etherToWei(1).plus(1);
+    const [platformShares, shares] = await platformTerms.calculateNeumarkDistribution(reward);
+
+    expect(shares.plus(platformShares)).to.be.bignumber.eq(reward);
+    expect(shares)
+      .to.be.bignumber.eq(etherToWei(0.5).plus(1))
```

Comment:
just please remember that not every nice decimal has nice binary representation... we however have an integer here via `etherToWei`